modules/tinymce/src/core/main/ts/api/dom/EventUtils.ts

```diff
@@ -205,26 +204,6 @@ const bindOnReady = function (win, callback, eventUtils) {
     }
   };
 
-  const waitForDomLoaded = function () {
-    if (isDocReady()) {
-      removeEvent(doc, 'readystatechange', waitForDomLoaded);
-      readyHandler();
-    }
-  };
-
-  const tryScroll = function () {
-    try {
-      // If IE is used, use the trick by Diego Perini licensed under MIT by request to the author.
-      // http://javascript.nwbox.com/IEContentLoaded/
-      doc.documentElement.doScroll('left');
-    } catch (ex) {
-      Delay.setTimeout(tryScroll);
-      return;
-    }
-
-    readyHandler();
-  };
-
```

Comment:
Deleted unused code.

Reply:
Can confirm, this was for old IE 10 or lower code and the rest got removed in 344122a0880ae60355dc6f9d8cb2ff813568a28f